BACKERS.md

```diff
@@ -0,0 +1,87 @@
+# Sponsors & Backers
+
+Material-UI is an MIT-licensed open source project. It's an independent project with ongoing development made possible thanks to the support of these awesome backers. If you'd like to join them, please consider:
+- [Become a backer or sponsor on Patreon](https://www.patreon.com/oliviertassinari).
+- [Become a backer or sponsor on OpenCollective](https://opencollective.com/material-ui).
+
+Through contributions, donations, and sponsorship, you allow Material-UI to thrive.
+Your donations directly support office hours, continued enhancements, great documentation and learning materials!
+
+### What's the difference between Patreon and OpenCollective?
+
+Funds donated via Patreon directly support [Olivier Tassinari](https://github.com/oliviertassinari)'s work on Material-UI.
+Funds donated via OpenCollective are managed transparently and will be used to compensate work and expenses by core team members.
+Your name/logo will receive proper recognition and exposure by donating on either platform.
```

Comment:
The wording of this paragraph could lead someone to think that funds donated to OpenCollective do not support you. I would suggest something like this:

Funds donated via Patreon directly support Olivier Tassinari's work on Material-UI. Funds donated via OpenCollective also support Olivier, but can also be shared amongst other contributors and pay for operating expenses. These funds are managed transparently through the OpenCollective website. Your name/logo will receive proper recognition and exposure by donating on either platform.

Reply:
> OpenCollective do not support you

Oh, I was assuming that it would morally be more right me not receiving OpenCollective donations by opening a Patreon page. But I like the idea :).